packages/react/package.json

```diff
@@ -33,6 +34,7 @@
     "@cds/core": "./dist/core",
     "@babel/core": "7.10.2",
     "@babel/preset-env": "7.10.2",
+    "@lit-labs/react": "^1.0.0",
```

Comment:
Do we need to have this listed as a devDependency if its in dependencies?

Reply:
Nope. I'll remove it when I squash for merge.